docs/source/performance/apq.md

```diff
@@ -150,16 +150,18 @@ How exactly this works depends on exactly which CDN you chose. Configure your CD
 
 ## Cache configuration
 
-Inside Apollo Server, the automated persisted query registry is stored in a user-configurable cache.  A local in-memory cache is enabled by default, however, if a general default `cache` has been provided as a top-level option to the `ApolloServer` constructor options, that cache will be used instead.
+By default, Apollo Server stores its APQ registry within its local in-memory cache. If you provide a different `cache` as a top-level option to the `ApolloServer` constructor, Apollo Server uses that cache instead.
 
-To override a the cache implementation specifically for automated persisted queries, leaving the general cache untouched, provide an instance of the desired implementation to the `ApolloServer` constructor option for `persistedQueries`.  The available classes and the npm modules they are available from are as follows:
+You can also designate a cache _specifically_ for the APQ registry. To do so, provide an instance of your preferred cache class to the `ApolloServer` constructor as the `persistedQueries` option. The following backing data stores are supported:
```

Comment:
This would need to indicate that the instance should be provided on the `cache` key inside of `persistedQueries`.  i.e.

```javascript
const server = new ApolloServer({
  persistedQueries: {
    cache: /* Instance here! */
  },
);
```

```suggestion
You can also designate a cache _specifically_ for the APQ registry. To do so, provide an instance of your preferred cache class to the `ApolloServer` constructor as a `cache` option nested inside the `persistedQueries` options object. The following backing data stores are supported:
```

**Or maybe**, we should actually provide this as an example right here?  (We make this more clear in the individual examples below, so this less than concrete example might be superfluous?)

```suggestion
You can also designate a cache _specifically_ for the APQ registry. To do so, provide an instance of your preferred cache class to the `ApolloServer` constructor as a `cache` option nested inside the `persistedQueries` options object:

```javascript
const server = new ApolloServer({
  typeDefs,
  resolvers,
  persistedQueries: {
    cache: /* Instance here! */
  },
);

The following backing data stores are supported:
```
```